iredis/entry.py

```diff
@@ -282,6 +288,8 @@ def repl(client, session):
 @click.option("-n", help="Database number.", default=None)
 @click.option("--raw/--no-raw", default=False, is_flag=True, help=RAW_HELP)
 @click.option("--decode", default=None, help=DECODE_HELP)
+@click.option("--version", help="Print iredis version", is_flag=True,
```

Comment:
Can we just use the builtin version support?

https://click.palletsprojects.com/en/7.x/api/#click.version_option
```suggestion
@click.version_option()
```

Reply:
update,